doc/openapi.yaml

```diff
@@ -19,7 +19,7 @@ paths:
         example: '1.0'
       responses:
         '200':
-          description: returns no need to update
```

Comment:
you don't have to commit this file, it gets automatically regenerated

Reply:
I included this file to update it

Reply:
this file is autogenerated in the `main` branch, check https://github.com/rootstrap/rails_api_base/blob/main/.github/workflows/ci.yml#L188-L195

Reply:
Yeah, was my mistake